index.html

```diff
@@ -69,7 +69,6 @@ <h2 class="connect">Let's connect</h2>
         </ul>
       </div>
     </section>
-
     <section class="projects">
       <div class="container-projects">
         <!--card one-->
```

Comment:
- [ ] You will notice the first card has the image on the left side, kindly adapt this in your project too so that it would be template-perfect, at the moment, yours is starting from the right.


### Figma
![image](https://github.com/DHE19/portfolio-microverse/assets/86200356/2b27cd8f-b5e3-4daf-8d04-6916e9d9866d)
### Your project
![image](https://github.com/DHE19/portfolio-microverse/assets/86200356/a09ed6b0-6e5b-42f9-bff8-2ff7a9c0d599)